msg/vehicle_command.msg

```diff
@@ -155,6 +155,11 @@ uint8 FAILURE_TYPE_SLOW = 5
 uint8 FAILURE_TYPE_DELAYED = 6
 uint8 FAILURE_TYPE_INTERMITTENT = 7
 
+uint8 SPEED_TYPE_AIRSPEED = 0
```

Comment:
```suggestion
uint8 DO_CHANGE_SPEED_TYPE_AIRSPEED = 0
```
To make it clear that it is linked to this command.

Reply:
@sfuhrer I'd rather introduce an enum in mavlink and name it accordingly, see e.g. https://mavlink.io/en/messages/ardupilotmega.html#SPEED_TYPE